scanner/src/main/kotlin/Scanner.kt

```diff
@@ -193,7 +193,7 @@ class Scanner(
 
         val vcsPathsForProvenances = getVcsPathsForProvenances(provenances)
 
-        val filteredScanResults = filterScanResultsByVcsPaths(controller.getAllScanResults(), vcsPathsForProvenances)
+        val scanResults = filterScanResultsByVcsPaths(controller.getAllScanResults(), vcsPathsForProvenances)
```

Comment:
commit-msg: Please use `refactor(scanner)` instead of `feat(model,scanner)`

Reply:
commit message body needs update too. Maybe simpliy say "Extract the padding of scan results to functions to prepare for an upcoming change."